awx/main/models/workflow.py

```diff
@@ -433,6 +433,14 @@ class Meta:
         default=None,
         on_delete=models.SET_NULL,
     )
+    job_template = models.ForeignKey(
+        'JobTemplate',
+        related_name='sharded_jobs',
+        blank=True,
+        null=True,
+        default=None,
+        on_delete=models.SET_NULL,
+    )
```

Comment:
I would prefer a field name that include any information about its function, like `shard_job_template`.

Reply:
we will need a view for the JT's `sharded_jobs` and linked in JT related